src/Rules/Conditions/TreeDescentCondition.cs

```diff
@@ -33,7 +33,7 @@ public override bool Matches(IA11yElement e)
 
         public override string ToString()
         {
-            return Invariant($"{this.ParentCondition} / {this.ChildCondition}");
+            return string.Format(CultureInfo.CurrentCulture, ConditionDescriptions.TreeDescentConditionFormat, this.ParentCondition, this.ChildCondition);
```

Comment:
@codeofdusk brought up an interesting point about the conditions. If we can successfully argue that they're used as a domain-specific language, then the would no longer be in scope for localization. A corollary of that might be that this becomes non-localizable and should use CultureInfo.InvariantCulture